### DISCUSSION TITLE
A loader was not found

### DISCUSSION THREAD

**Discussion by CCCDDDEEEFFF** [pallets/jinja#1949] [Q&A]
I am trying to load a package in a docker container but for some reason it is not able to find the loader. loaders.py is in the folder where is says it is but it is not able to find it. 

File "/usr/local/lib/python3.6/dist-packages/edalize/edatool.py", line 183, in __init__
    loader=PackageLoader(__package__, "templates"),
  File "/usr/local/lib/python3.6/dist-packages/jinja2/loaders.py", line 291, in __init__
    assert loader is not None, "A loader was not found for the package."
AssertionError: A loader was not found for the package.

Environment:

- Python version: 3.6
- Jinja version: 3.1.3

**Comment by kwist-sgr** [comment]
I guess this loader isn't jinja's, but [importlib's](https://docs.python.org/3/library/importlib.html#importlib.util.find_spec)


[`PackageLoader.__init__`](https://github.com/pallets/jinja/blob/9c6c3198996014756c111ac4d2e80b0423b3b95a/src/jinja2/loaders.py#L294:L298)
